modules/statistics-writer-workflowoperation/src/main/resources/OSGI-INF/operations/statisticswriter.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0"
```

Comment:
I feel like statistics-writer-woh.xml would be a better name.

Reply:
It's more consistent, yes. I'll change it.